Ginger/GingerCore/ActOcr.cs

```diff
@@ -331,12 +331,14 @@ private void ExecuteImageOperation()
 
         private void ExecutePdfOperation()
         {
+            string password = ValueExpression.Calculate(OcrPassword);
```

Comment:
OcrPassword is stored on InputValues, hence it will be already decrypted, hence no need to evaluate or decrypt again. Please check InputValues on Act object

Reply:
![OCRPassword](https://github.com/Ginger-Automation/Ginger/assets/58476741/ae8fd64f-38f8-4f69-98a3-3017e4ad744d)

As you can see, in above Image the OCRPassword isn't already decrypted. That's why decided to use the EncryptionHandler library.